src/Components/MoodRating/MoodRating.js

```diff
@@ -0,0 +1,150 @@
+import React, { useState, useContext } from 'react'
+import style from './MoodRating.module.scss'
+import { ViewContext } from '../../Context/ViewContext'
+import { SessionContext } from '../../Context/SessionContext'
+import { postSession } from '../../apiCalls'
+import moodIcon1 from '../../assets/moodIcons/moodIcon1.png'
+import moodIcon2 from '../../assets/moodIcons/moodIcon2.png'
+import moodIcon3 from '../../assets/moodIcons/moodIcon3.png'
+import moodIcon4 from '../../assets/moodIcons/moodIcon4.png'
+import moodIcon5 from '../../assets/moodIcons/moodIcon5.png'
+
```

Comment:
I wonder if this could be converted into an `import * from '../../assets/moodIcons'`? I am not sure about that though. No need to make these changes, I'm just thinking out loud!